.circleci/config.yml

```diff
@@ -0,0 +1,110 @@
+version: 2
+
+aliases:
+
+  - defaults: &defaults
+      docker:
+        - image: circleci/node:carbon
+      working_directory: ~/code
+  - &restore-repo-cache
+      key: shared-goodcity-repo-{{ .Environment.CIRCLE_SHA1 }}
+  - &restore-yarn-cache
+      keys:
+        - shared-goodcity-yarn-packages-{{ checksum "yarn.lock" }}
+  - &save-yarn-cache
+      key: shared-goodcity-yarn-packages-{{ checksum "yarn.lock" }}
+      paths:
+        - node_modules
+        - ~/.cache/yarn
+  - &yarn
+    |
+      yarn install --non-interactive --cache-folder ~/.cache/yarn --frozen-lockfile
+  - &restore-bower-cache
+      key: shared-goodcity-bower-components-{{ checksum "bower.json" }}
+  - &save-bower-cache
+      key: shared-goodcity-bower-components-{{ checksum "bower.json" }}
+      paths:
+        - bower_components
+
+jobs:
+  checkout_code:
+    <<: *defaults
+    steps:
+      - checkout
+      - save_cache:
+          key: shared-goodcity-repo-{{ .Environment.CIRCLE_SHA1 }}
```

Comment:
@Darshan-Chauhan  Is it worth moving this key up into the aliases section next to where it is already defined? (Small detail)